package.json

```diff
@@ -1,20 +1,19 @@
 {
   "name": "mercurius",
   "version": "11.3.0",
-  "description": "Fastify GraphQL adapter with gateway and subscription support",
+  "description": "Fastify GraphQL adapter with subscription support",
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
     "docs": "docsify serve",
-    "unit": "tap test/*.js test/gateway/*.js test/internals/*.js",
-    "cov": "tap  --coverage-report=html -J test/*.js test/gateway/*.js",
+    "unit": "tap test/*.js test/internals/*.js",
+    "cov": "tap --jobs=1 --coverage-report=html test/*.js test/internals/*.js",
```

Comment:
why?

Reply:
that's an error I needed to run to check what test was covering a specific code and forgot to remove it.